elasticsearch-vmss/azuredeploy.json

```diff
@@ -0,0 +1,504 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "type": "string",
+      "defaultValue": "es-azure",
+      "metadata": {
+        "description": "The name of the cluster."
+      }
+    },
+    "vmSku": {
```

Comment:
Sure it's needed. The sku makes a big difference in the cluster, and very much depends on how the customer plans on using it.